GDJS/Runtime/pixi-renderers/pixi-filters-tools.ts

```diff
@@ -88,7 +88,7 @@ namespace gdjs {
     /** A wrapper allowing to create a PIXI filter and update it using a common interface */
     export type FilterCreator = {
       /** Function to call to create the filter */
-      makePIXIFilter: (layer: gdjs.Layer, effectData: EffectData) => any;
+      makePIXIFilter: (layer: gdjs.Layer | gdjs.RuntimeObject, effectData: EffectData) => any;
```

Comment:
I guess this is def wrong, I'll check each effect later, to be sure that there are no errors.

Reply:
Yeah we probably actual need to either remove this first argument from all filters, or rename it according to what it's used for... no big deal for now.